buffer/src/test/java/io/netty/buffer/AbstractByteBufTest.java

```diff
@@ -2281,10 +2281,12 @@ public void run() {
                         byte[] array = new byte[8];
                         buf.readBytes(array);
 
+                        //noinspection MisorderedAssertEqualsArguments
```

Comment:
yah remove the suppression ... seems like a IntelliJ bug based upon the description.